doc/source/whatsnew/v1.6.0.rst

```diff
@@ -131,7 +131,7 @@ Timezones
 
 Numeric
 ^^^^^^^
--
+- Bug in :meth:`DataFrame.add` array ufuncs with more than 2 inputs fails or more  (:issue:`39853`)
```

Comment:
Please clarify. This does not explain the bug properly. You have to include at least that mixed Series and DataFrame inputs are concerned